repos.d/rpm/rosa.yaml

```diff
@@ -60,7 +63,8 @@
   color: '3757a1'
   minpackages: 6500
   sources:
-    - name: [ main/release, main/testing, main/updates, non-free/release, non-free/testing, non-free/updates, restricted/release, restricted/testing, restricted/updates, contrib/release, contrib/testing, contrib/updates ]
+    # see comments bellow in rosa2019.05
```

Comment:
Please duplicate the comment here, as repository entries are managed independently

Reply:
Done